browser-extension/mv2/src/client/js/scriptRuleHandler.js

```diff
@@ -128,24 +128,105 @@ RQ.ScriptRuleHandler.includeJSScriptsInOrder = function (scripts, callback, inde
 };
 
 RQ.ScriptRuleHandler.includeJS = function (script, callback) {
+  if (!script.value) throw new Error("Script value is empty");
```

Comment:
Instead of throwing an error, should we do `console.error` and add a more human-readable log like `Script value used in Requeslty Inject Script Rule is empty.

We should make sure, it doesn't break the webpage just that the rule isn't run.